packages/website/src/theme/index.js

```diff
@@ -64,7 +64,20 @@ export default {
       `,
     },
   },
+  palette: {
+    primary: '#102e4e',
+    secondary: '#e37700',
+  },
   Button: {
+    styles: {
+      base: css`
+        min-width: 160px;
```

Comment:
I've noticed this `min-width` affects tags closing buttons, I hope configuring tag's theme overwrites it
http://dev-style.localz.io/form/select-menu/#with-tags

Reply:
nice one @harsilspatel , i believe the `SelectMenu` component is by default using the button style for the closing button, but we can configure it with `SelectMenu.Button.styles.base`
https://bumbag.style/form/select-menu/#theming